app/src/main/java/com/xpay/kotlinutils/XpayUtils.kt

```diff
@@ -17,15 +17,20 @@ object XpayUtils {
     var apiKey: String? = null
     var communityId: String? = null
     var variableAmountID: Number? = null
+
     // Payment methods data
     var PaymentOptionsTotalAmounts: PaymentOptionsTotalAmounts? = null
         private set
     var payUsing: PaymentMethods? = null
+
+    // Pay request body
+    private var body: RequestBody? = null
```

Comment:
i think it's better to move this inside the pay method, as there is no need for it to be a global member inside our Object.